src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -627,6 +626,32 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _State.Profile().ColorSchemeName(val.Name());
     }
 
+    bool Profiles::IsBellStyleFlagSet(const uint32_t flag)
+    {
+        return (WI_EnumValue(_State.Profile().BellStyle()) & flag) == flag;
```

Comment:
Wait uh, what exactly is happening here? Is this just `WI_IsFlagSet`?

Reply:
or `WI_AreAllFlagsSet`?

Reply:
explained offline -- this is because the WI macros _require_ a compile-time constant. We have to do it manually.